ga4gh/static/swagger.json

```diff
@@ -0,0 +1,172 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "GA4GH Reference Server",
+    "version": "0.2.0",
+    "title": "GA4GH Reference Server",
+    "contact": {
+      "email": "someone@ucsc.edu"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "localhost:8000",
+  "tags": [
+    {
+      "name": "variants",
+      "description": "Variant endpoints",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "documentation site"
+      }
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/variants/{id}": {
+      "get": {
+        "tags": [
+          "variants"
+        ],
+        "description": "Returns variant for a given ID",
+        "summary": "Find variants by ID",
+        "operationId": "getVariant",
+        "produces": [
```

Comment:
I am wondering if we should wait to implement the auto generation of the documentation until we move to protocol buffers. I am expecting that manual implementation will not be too painful if the schema doesn't change very often or very much. Once we have protocol buffers then we can connect it in to be automatically generated from that documentation layer.